collect_app/src/main/java/org/odk/collect/android/injection/config/AppDependencyModule.java

```diff
@@ -635,4 +642,9 @@ public ImageCompressionController providesImageCompressorManager() {
     public FormLoaderTask.FormEntryControllerFactory formEntryControllerFactory(SettingsProvider settingsProvider) {
         return new CollectFormEntryControllerFactory();
     }
+
+    @Provides
+    public SavepointFinder providesSavepointFinder() {
```

Comment:
If this isn't being overridden in `FormUriActivity` (as another comment suggests), then you don't need to be injected with Dagger.

Reply:
Done.